src/reducers/icons.js

```diff
@@ -1,4 +1,11 @@
-import Immutable, { Map } from 'immutable'
+import { Map } from 'immutable'
+
+function updateOrAddVm ({ state, payload: { icons } }) {
```

Comment:
@mareklibra are you really sure you want to call this method `updateOrAddVm`? Seems really icon-centric. If this is a result of a copy-paste from the VM equivalent probably that is a sign that we want to extract this to a separate helper called e.g. `mergeById` and call it from both reducers?